### ISSUE TITLE
Hey, its here! Hacktober Fest 2021

### ISSUE THREAD

**Issue by abhilashmnair** [abhilashmnair/HacktoberFest2021#1]

**Comment by KamalAres** [comment]
Can I add Basic Java Programs like Fibonacci, palindrome in Java Folder? Can you assign this task to me?

**Comment by abhilashmnair** [comment]
Of course you can! But try to commit some intermediate level codes if you can!

**Comment by akash9891** [comment]
I used to give competitive programming contests and here i want to know what can i do to contribute in cpp?

**Comment by naveenks2002** [comment]
Please can u assign me this task . I am good at Data structures and Algorithms

**Comment by shivani0126** [comment]
I want to contribute to this issue, can I add Sorting programs in Java(like Bubbble,Selection,Heap,Merge and Insertion Sort)

**Comment by abhilashmnair** [comment]
> I want to contribute to this issue, can I add Sorting programs in Java(like Bubbble,Selection,Heap,Merge and Insertion Sort)

Sure you can. Make sure each of your code is having different pull requests.

**Comment by abhilashmnair** [comment]
> I used to give competitive programming contests and here i want to know what can i do to contribute in cpp?

Contribute some intermediate or interesting codes that would encourage beginners to contribute to open source. 😊

**Comment by isha614** [comment]
I am a competitive programmer, coder.  Can I contribute some good coding ques in cpp folder?

**Comment by gokul5102** [comment]
I wish to contribute to this repository.Can I add some backtracking/array programs in cpp?

**Comment by Amitlpu-840** [comment]
how many contributions can I make to this repository? are there any limitations?

**Comment by riya244** [comment]
checkout pull request #79 @KamalAres

**Comment by abhilashmnair** [comment]
> checkout pull request #79 @KamalAres

Marked Invalid. Violated _Rules_

**Comment by abhilashmnair** [comment]
> I am a competitive programmer, coder. Can I contribute some good coding ques in cpp folder?

Definitely! You can create a folder named **Competetive Programming** and add your code. All the best!

**Comment by abhilashmnair** [comment]
> I wish to contribute to this repository.Can I add some backtracking/array programs in cpp?

Sure!

**Comment by abhilashmnair** [comment]
> how many contributions can I make to this repository? are there any limitations?

There is no limit! But definitely DON'T SPAM!

**Comment by YushChaudhary** [comment]
Hey abhilash can you check my two commit's #89 #90

**Comment by rosetanya898** [comment]
Can you look into my pull request #146
